Makefile

```diff
@@ -149,6 +149,20 @@ ifeq (, $(shell which yamlfmt))
 endif
 	yamlfmt -conf tools/.yamlfmt .
 
+# Tools
+
+.PHONY: install-lazyfs
+install-lazyfs: bin/lazyfs
+
+bin/lazyfs:
+	rm /tmp/lazyfs -rf
+	git clone https://github.com/dsrhaslab/lazyfs /tmp/lazyfs
+	cd /tmp/lazyfs/; git checkout 94ef5e60117f2a6c6d12b29e09e287c3893150ca
```

Comment:
Could we refer to a specific tag here to be a bit more human readable?

Alternatively we could potentially just clone the specific tag so no separate checkout command is required.

Reply:
There were not recent tags when I wrote the PR.

However, last week there was a new release, so there is a tag for us to use!